### PR TITLE
Class 01 init

### PR DESCRIPTION
**Pull request by grandquista**

### CODE REVIEW

**Comment by Patricia888** [comment]
- [x] Files names and branch names match with specifications
- [x] Partner is a collaborator on repo
- [x] Program prints an intro and menu to user
- [x] Restaurant's menu includes appetizers, entrees, desserts, and beverages, 3 in each category
- [x] Program prompts the user for an order
- [x] When a user enters an item, the program prints an acknowledgment of their input
- [ ] Program tells the user how to exit
- [x] Wrote README

File: snakes_cafe.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+
+def menu_display():
+    print('Appetizers')
+    print('----------')
+    for item, properties in MENU.items():
+        if properties['categories'] == 'appetizers':
+            print(item.title())
+    print()
```
Comment (Woojgh) [inline snakes_cafe.py]: Instead of empty print statements, you could use the new line option ( \n ).

File: snakes_cafe.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        else:
+            order[user_request] = order.get(user_request, 0) + 1
+            print(ORDER_RESPONSE.format(order[user_request], user_request))
+
+
+if __name__ == '__main__':
+    try:
+        main()
+    except KeyboardInterrupt:
+        print('quit should be used to exit cleanly')
```
Comment (Woojgh) [inline snakes_cafe.py]: Great try except!

File: snakes_cafe.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+
+def main() -> None:
+    """main."""
+    print(INSTRUCTIONS_HEADER)
+    menu_display()
+    order: Dict[str, int] = {}
+    while True:
+        user_request = input(USER_INPUT_REQUEST)
+        user_request = user_request.strip().lower()
```
Comment (Woojgh) [inline snakes_cafe.py]: Nice strip!

**Pull request merged**